src/components/about/AboutUs.tsx

```diff
@@ -2,6 +2,26 @@ const AboutUs = () => {
   return (
```

Comment:
1. Run `git pull` on this branch, some changes were made. 
2. Add a white border, round the corners if you'd like.
3. Increase the pixels on line 16 to match the design.